crates/bitwarden-exporters/src/cxf/login.rs

```diff
@@ -56,10 +56,21 @@ pub(super) fn to_login(
     totp: Option<&TotpCredential>,
     scope: Option<&CredentialScope>,
 ) -> Login {
+    // Use basic_auth username first, fallback to non-empty passkey username
+    let username = basic_auth
+        .and_then(|v| v.username.clone().map(|v| v.into()))
+        .or_else(|| passkey.and_then(|p| (!p.username.is_empty()).then_some(p.username.clone())));
```

Comment:
I think this is a bit more readable.
```suggestion
    let username = basic_auth
        .and_then(|v| v.username.clone().map(Into::into))
        .or_else(|| {
            passkey
                .filter(|p| !p.username.is_empty())
                .map(|p| p.username.clone())
        });
```